### PR TITLE
[MNOE-767] - Display aborted tooltip in dashboard

### PR DESCRIPTION
**Pull request by fgourichon**
Tooltip to explain subscription 'Aborted' status was not displayed in details page in dashboard.
Also resized columns in subscriptions list

### CODE REVIEW

File: src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.controller.coffee [hunk 1]
```diff
@@ -0,0 +1,9 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('SubscriptionInfoController', ($filter, $stateParams, $log, $uibModalInstance) ->
```
Comment (alexnoox) [inline src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.controller.coffee]: Remove unused services

File: src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div id="subscription-info">
+  <div class="modal-body" translate>mno_enterprise.templates.dashboard.provisioning.subscriptions.status.info.body.aborted</div>
+  <div class="modal-footer">
+    <div class="row actions">
+      <div class="col-xs-12">
+        <a href="" ng-click="vm.onCancel()"><i class="fa fa-times"></i>{{ 'mno_enterprise.templates.dashboard.provisioning.subscriptions.status.info.close' | translate}}</a>
```
Comment (alexnoox) [inline src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.html]: Add a space or `&nbsp;` between the `i` and the translation

File: src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div id="subscription-info">
+  <div class="modal-body" translate>mno_enterprise.templates.dashboard.provisioning.subscriptions.status.info.body.aborted</div>
```
Comment (alexnoox) [inline src/app/views/provisioning/subscription-status-info-modal/subscription-status-info.html]: Should we also display a `.modal-header`?

File: src/app/views/provisioning/subscription.html [hunk 4]
```diff
@@ -39,7 +39,10 @@ <h3 class="text-color" translate>mno_enterprise.templates.dashboard.provisioning
 
           <div class="form-group">
             <label class="control-label text-color" translate>mno_enterprise.templates.dashboard.provisioning.subscription.status</label>
-            <input class="form-control hide-border" ng-model="vm.subscription.status" readonly/>
+            <div>
```
Comment (alexnoox) [inline src/app/views/provisioning/subscription.html]: You should not need this `div`.

**Comment by fgourichon** [comment]
All fixed @alexnoox

**Pull request merged**